### PR TITLE
Released new version

### PR DESCRIPTION
**Pull request by malikalhack**
Fixed bugs with hardcode

### CODE REVIEW

**Review by malikalhack2 — CHANGES_REQUESTED** [review]
Replace the cast type on line 31 of the src/vl_queue.c file.

File: src/vl_queue.c [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
  /****************************** Included files ********************************/
 #include "common.h"
 #include <stdlib.h>
 #include <malloc.h>
 /****************************** Private  variables ****************************/
-static uint8_t *queue;
+static data_t *queue;
 static uint8_t length;
 static volatile uint8_t head;
 static volatile uint8_t tail;
```
Comment (malikalhack2) [inline src/vl_queue.c]: Redefine queue and stack index types.

**Review by malikalhack2 — CHANGES_REQUESTED** [review]

**Review by malikalhack2 — CHANGES_REQUESTED** [review]
Correct all types of incoming and outgoing data.

File: src/vl_queue.c (hunk 1 above)
Comment (malikalhack) [reply src/vl_queue.c]: Done!

**Review by malikalhack2 — APPROVED** [review]
Looks good

**Pull request merged**